.changeset/orange-rings-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/webchannel-wrapper": patch
+---
+
+fix: Fix a typo in the webchannel-wrapper's package.json that affected ems5 exports.
```

Comment:
drive-by comment: there is a typo in the release notes: "ems5" should be "esm5". That is proving to be a difficult acronym to spell correctly.

Reply:
@ehsannas

Reply:
ouch :(

It seems that this doesn't show up on the release notes (https://firebase.google.com/support/release-notes/js) page though (because it's a webchannel-wrapper package change?)

Reply:
Hmm not sure about webchannel-wrapper release notes showing up. Maybe they're not included, but, rather, expect to be mentioned by the dependents upgrading their dependency (i.e. a Firestore change that has its own release note that goes with the webchanne-wrapper upgrade).

Reply:
Hi, I wrote the release notes for the 10.12.3 release where this went out. I mistakingly excluded this from the release notes. Looking at older release notes, we list changes to `webchannel-wrapper` under Firestore.

Sorry if I caused any confusion- I'll create a CL to mention this change in the release notes.

Reply:
Please add me as reviewer for your CL. Thanks :)

Reply:
> Please add me as reviewer for your CL. Thanks :)

Sounds good. I added you as a reviewer